client/signup/steps/woocommerce-install/step-business-info/index.tsx

```diff
@@ -0,0 +1,226 @@
+import { getSiteDomain } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import SupportCard from '../components/support-card';
+import { ActionSection, StyledNextButton } from '../confirm';
+import { useSiteSettings, WOOCOMMERCE_ONBOARDING_PROFILE } from '../hooks/use-site-settings';
+import type { WooCommerceInstallProps } from '..';
+import './style.scss';
+
+export default function StepBusinessInfo( props: WooCommerceInstallProps ): ReactElement | null {
+	const { goToNextStep, isReskinned } = props;
+	const { __ } = useI18n();
+
+	const dispatch = useDispatch();
+	const siteId = useSelector( getSelectedSiteId ) as number;
+	const siteDomain = useSelector( ( state ) => getSiteDomain( state, siteId ) ) as string;
+
+	const { get, save, update } = useSiteSettings( siteId );
+
+	function updateProductTypes( type: string ) {
+		updateOnboardingProfile( 'product_types', type );
+	}
+
+	function updateProductCount( count: string ) {
+		updateOnboardingProfile( 'product_count', count );
+	}
+
+	function updateSellingVenues( venue: string ) {
+		updateOnboardingProfile( 'selling_venues', venue );
+	}
+
+	function updateOtherPlatform( platform: string ) {
+		updateOnboardingProfile( 'other_platform', platform );
+	}
+
+	function updateOtherPlatformName( name: string ) {
+		updateOnboardingProfile( 'other_platform_name', name );
+	}
+
+	function updateOnboardingProfile( key: string, value: string | boolean ) {
+		const onboardingProfile = get( WOOCOMMERCE_ONBOARDING_PROFILE ) || {};
+
+		const updatedOnboardingProfile = {
+			...onboardingProfile,
+			[ key ]: value,
+		};
+
+		update( WOOCOMMERCE_ONBOARDING_PROFILE, updatedOnboardingProfile );
+	}
+
+	function getProfileValue( key: string ) {
+		const onboardingProfile = get( WOOCOMMERCE_ONBOARDING_PROFILE ) || {};
+
+		return onboardingProfile[ key ] || '';
+	}
+
+	function getContent() {
+		return (
+			<>
+				<div className="step-business-info__info-section" />
+				<div className="step-business-info__instructions-container">
+					<SelectControl
+						label={ __( 'What type of products will be listed? (optional)', 'woocommerce-admin' ) }
+						value={ getProfileValue( 'product_types' ) }
+						options={ [
+							{ value: '', label: '' },
+							{ value: 'physical', label: __( 'Physical Products', 'woocommerce-admin' ) },
+							{ value: 'downloads', label: __( 'Downloads', 'woocommerce-admin' ) },
+							{ value: 'subscriptions', label: __( 'Subscriptions', 'woocommerce-admin' ) },
+						] }
+						onChange={ updateProductTypes }
+					/>
+
+					<SelectControl
+						label={ __(
+							'How many products do you plan to display? (optional)',
+							'woocommerce-admin'
+						) }
+						value={ getProfileValue( 'product_count' ) }
+						options={ [
+							{ value: '', label: '' },
+							{ value: '0', label: __( "I don't have any products yet.", 'woocommerce-admin' ) },
```

Comment:
```suggestion
							{ value: '0', label: __( "I don't have any products yet." ) },
```

We shouldn't be using woo's text domain in calypso. Translation memory will catch these and move them over as needed.

Reply:
Thanks! Removed this and all other instances of woo's text domain.